lib/helper/WebDriver.js

```diff
@@ -2055,6 +2055,17 @@ class WebDriver extends Helper {
     return this.browser.getGeoLocation();
   }
 
+  /**
+   * {{> grabElementSize }}
+   *
+   */
+  async grabElementSize(locator, prop) {
+    const res = await this._locate(locator, true);
+    assertElementExists(res, locator);
+
+    return forEachAsync(res, async el => el.getSize(prop));
```

Comment:
Webdriver return only width and height, but Puppeteer method return x, y, width and height.
Can you make the same return values?

Reply:
Sure, not a problem :)

Reply:
@Vorobeyko thanks bro for your review! I pushed the changes!